optaweb-employee-rostering-backend/src/main/resources/application-production.properties

```diff
@@ -13,14 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-
 # Datasource
-spring.datasource.url=jdbc:postgresql://${DATABASE_HOST:postgresql}:5432/${DATABASE_NAME}
-spring.datasource.username=${DATABASE_USER}
-spring.datasource.password=${DATABASE_PASSWORD}
-spring.jpa.hibernate.ddl-auto=update
-# https://stackoverflow.com/questions/43905119/postgres-error-method-org-postgresql-jdbc-pgconnection-createclob-is-not-imple
-spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation=true
-
-# Disable H2 console enabled in the default profile
-spring.h2.console.enabled=false
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.jdbc.url=jdbc:postgresql://${DATABASE_HOST:postgresql}:5432/${DATABASE_NAME}
+quarkus.datasource.username=${DATABASE_USER}
+quarkus.datasource.password=${DATABASE_PASSWORD}
+quarkus.hibernate-orm.database.generation=update
```

Comment:
I think this file is unused. Quarkus only reads `application.properties`. Please confirm and remove this file.